package.json

```diff
@@ -26,7 +26,16 @@
     "test:changes": "yarn test --changedSince master --verbose=true",
     "test:changes:watch": "yarn test --changedSince master --watch --verbose=true",
     "test:watch": "yarn test --watch",
-    "test:update-snaps": "yarn test -u"
+    "test:update-snaps": "yarn test -u",
+    "pretest:ci": "rm -rf .nyc_output || true",
+    "posttest:ci": "yarn report:combined",
+    "mkdir:reports": "mkdir reports || true",
+    "precopy:reports": "yarn mkdir:reports",
+    "copy:reports": "cp cypress-coverage/coverage-final.json reports/from-cypress.json && cp jest-coverage/coverage-final.json reports/from-jest.json",
+    "precombine:reports": "yarn copy:reports && mkdir .nyc_output || true",
+    "combine:reports": "nyc merge reports && mv coverage.json .nyc_output/out.json",
```

Comment:
fuck yeah

Reply:
The credit goes to the wonderful people at cypress. Mostly copy and paste with few modifications to suit our case 😃